_config.yml

```diff
@@ -79,33 +79,33 @@ rockstarSpeakersButtonText: "See all speakers"
 
 # Location Block
 locationBlockTitle: "Location"
-eventLocationName: "Conference Hall"
-eventLocationLink: "http://cabinet.kinopalace.lviv.ua/showtimes/"
-eventStreetAddress: "22 Teatral'na St"
-eventAddressLocality: "Lviv"
-eventAddressLocalityRegion: "Lvivska"
-eventPostalCode: "79000"
-eventAddressCountry: "Ukraine"
-eventStartTime: "2014-10-25T10:00"
+eventLocationName: "Columbia University, Uris Hall"
+eventLocationLink: "www8.gsb.columbia.edu"
+eventStreetAddress: "3022 Broadway"
+eventAddressLocality: "New York"
+eventAddressLocalityRegion: "New York"
+eventPostalCode: "10027"
+eventAddressCountry: "U.S"
+eventStartTime: "2019-02-15T00:00"
```

Comment:
Taylor said that the conference is only one day now. February 15th. 9:30AM-8:30PM